data/_models/shared/Agent.csv

```diff
@@ -1,4 +1,4 @@
 tableName,tableExtends,columnName,columnType,key,required,refSchema,refTable,refLink,refBack,validation,semantics,description,profiles
-Agent,,,,,,,,,,,"http://xmlns.com/foaf/0.1/Agent","An agent (eg. person, group, software or physical artifact).","DCAT,Beacon v2"
```

Comment:
I think we should merge contact and organisation(unit) into agent with subclasses